sdf/final-project/presentation.md

```diff
@@ -10,12 +15,15 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
 - [ ] **Introduction**: Does the presentation start with an introduction covering previous work/life experiences and connection to the problem?
 - [ ] **Target User**: Does the presentation clearly identify the target user?
 - [ ] **Target Problem**: Does the presentation clearly identify the problem the project addresses?
-- [ ] **Market Research and Problem Validation**: Is there evidence of market research or validation efforts, such as discussions with potential users, references to previous work/life experiences, and quanitative analysis?
+<!-- (at least 2/3) -->
+- [ ] **Market Research and Problem Validation**: Is there evidence of market research or validation efforts, such as discussions with potential users, references to previous work/life experiences, and quanitative analysis? 
 - [ ] **Solution Appeal and Usability**: Is the proposed solution presented in an engaging and compelling manner, making it easy to envision real users benefiting from and engaging with the application?
-- [ ] **Professional Preparation**: Is there a clear connection between the project work and the skills or experiences that prepare trainees for professional roles, potentially linking to industry categories or needs (e.g., as outlined on https://rfp.dpi.dev)?
+<!-- maybe make professional preparation it's own section -->
+- [ ] **Professional Preparation**: Is there an explicit connection between the project work and the skills or experiences that prepare trainees for professional roles, potentially linking to industry categories or needs (e.g., as outlined on https://rfp.dpi.dev), or connecting to a real business?
```

Comment:
This sentence is a bit confusing from a trainee's perspective. When I read it, I am unsure what the first part means. Suggestion: "Does the trainee explicitly connect their project work, skills they've developed, or professional experiences to the industry needs (e.g., as outlined on https://rfp.dpi.dev), or a real business/company partner?"